service/docs/source/runtime.rst

```diff
@@ -95,12 +103,51 @@ Runtime launch options.
     $ # show all options and available launchers
     $ geopmlaunch --help
 
-  The `GEOPM Runtime tutorial
-  <https://github.com/geopm/geopm/tree/dev/tutorial#geopm-tutorial>`_ shows how
-  to profile unmodified applications, select and evaluate different GEOPM Agent
-  algorithms, and how to add markup to an application.  The tutorial provides a
-  starting point for someone trying to get familiar with the GEOPM Runtime.
 
+.. admonition:: Quick start for Non-MPI applications
+
+   In order to profile non-MPI applications, the recommended approach is to launch
+   the GEOPM runtime alongside the target application. This can be done by launching
+   the ``geopmctl`` application in the background on every host node that the non-MPI 
+   application is expected to run. After a clean or forced termination of the 
+   application being profiled, the ``geopmctl`` application is designed to terminate
+   and generate a summarizing report file, and optionally, a time-series trace per host. 
+ 
+   The following requirements must be met while launching ``geopmctl`` with your 
+   non-MPI application:
+ 
+   * Both the ``geopmctl`` process and the application process must have
+     the ``GEOPM_PROFILE`` environment variable set to the **same**
+     value.
+   * The application process must have ``LD_PRELOAD=libgeopm.so.2`` set
+     in the environment, or the application binary must be linked
+     directly to ``libgeopm.so.2`` at compile time.
+   * The ``GEOPM_REPORT`` environment variable must be set in the
+     environment of the ``geopmctl`` process.
+   * The ``GEOPM_PROGRAM_FILTER`` environment variable can optionally be set to
+     explicitly list the program invocation name of the non-MPI processes that need
+     to be profiled.
```

Comment:
Maybe add a note mentioning why this option is useful (since it's optional), to avoid profiling any extraneous processes, or maybe adding a note similar to the one in the `geopmlaunch` documentation.